dts/arm/nxp/nxp_rt7xx_cm33_cpu0.dtsi

```diff
@@ -800,6 +800,39 @@
 		#size-cells = <0>;
 		status = "disabled";
 	};
+
+	mrt0: mrt@2d000 {
+		compatible = "nxp,mrt";
+		reg = <0x2d000 0x100>;
+		interrupts = <2 0>;
+		num-channels = <4>;
+		num-bits = <24>;
+		clocks = <&clkctl2 MCUX_MRT_CLK>;
+		resets = <&rstctl0 NXP_SYSCON_RESET(3, 26)>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		mrt0_channel0: mrt_channel@0 {
+			compatible = "nxp,mrt-channel";
+			reg = <0>;
+			status = "disabled";
+		};
+		mrt0_channel1: mrt_channel@1 {
```

Comment:
inconsistent file spacing

Reply:
What part of this is inconsistent? These mrt0_channel<n> are child to the mrt0.

Reply:
e.g. lines 783-804 have newline between entries, lines 815-830 do not

Reply:
I'm not sure what the standard is here but in the rest of the file, the instances of subnodes do not have newline; 291-312, 326-347, 361-382, etc... 

So do we want to have newline added to all of the rest of these?

Reply:
Subnodes at the top *do* have newlines https://github.com/zephyrproject-rtos/zephyr/blob/cdd1ca8a87ebf206997ee087958dc5d1fe4c9224/dts/arm/nxp/nxp_rt7xx_cm33_cpu0.dtsi#L38 it's consistency that should be there so if there's newlines then there should be newlines for all

Reply:
Ideally there should be one but it's outside the scope of this PR, just needs alignment here for what is being added, can fix the rest of the file in a future PR

Reply:
doing a future PR is just a waste of time. I've updated Lucian's PR